### PR TITLE
Optimizing generation of .inv.out files by only running daikon.PrintInvariants on class level and extracting method level invariants using python

### PR DESCRIPTION
**Pull request by sandervalstar**

### CODE REVIEW

File: getty/center.py [hunk 1]
```diff
@@ -144,18 +145,27 @@ def seq_get_invs(target_set_index_pair, java_cmd, junit_torun, go, this_hash, co
...
         current_count = 0
         total_count = len(target_set)
-    
+
     all_to_consider = set(target_set)
+    print "all_to_consider: ", all_to_consider
     if config.class_level_expansion:
         all_to_consider = (all_to_consider | expansion)
-    
+    print "all_to_consider: ", all_to_consider
```
Comment (Meggolbek) [inline getty/center.py]: Can we remove these extra prints here?

**Review by Meggolbek — COMMENTED** [review]
Looks good, except maybe extra print statemts, commented on these.

File: getty/center.py (hunk 1 above)
Comment (sandervalstar) [reply getty/center.py]: yes I can remove some print statements

**Comment by Meggolbek** [comment]
You can merge now!

**Review by Meggolbek — APPROVED** [review]

**Pull request merged**